integration-test/src/main/java/com/scalar/db/api/DistributedStorageSinglePartitionKeyIntegrationTestBase.java

```diff
@@ -154,59 +154,53 @@ public void getAndScanAndDelete_ShouldBehaveCorrectly() throws ExecutionExceptio
       random.setSeed(seed);
 
       truncateTable(partitionKeyType);
-      List<Value<?>> partitionKeyValues = prepareRecords(partitionKeyType);
+      List<Column<?>> partitionKeyValues = prepareRecords(partitionKeyType);
 
       String description = description(partitionKeyType);
 
       // for get
-      for (Value<?> partitionKeyValue : partitionKeyValues) {
+      for (Column<?> partitionKeyValue : partitionKeyValues) {
         // Arrange
         Get get = prepareGet(partitionKeyType, partitionKeyValue);
 
         // Act
-        Optional<Result> result = storage.get(get);
+        Optional<Result> optResult = storage.get(get);
 
         // Assert
-        assertThat(result).describedAs(description).isPresent();
-        assertThat(result.get().getValue(PARTITION_KEY).isPresent())
-            .describedAs(description)
-            .isTrue();
-        assertThat(result.get().getValue(PARTITION_KEY).get())
+        assertThat(optResult).describedAs(description).isPresent();
+        Result result = optResult.get();
+        assertThat(result.contains(PARTITION_KEY)).describedAs(description).isTrue();
+        assertThat(result.getColumns().get(PARTITION_KEY))
             .describedAs(description)
             .isEqualTo(partitionKeyValue);
-        assertThat(result.get().getValue(COL_NAME).isPresent()).describedAs(description).isTrue();
-        assertThat(result.get().getValue(COL_NAME).get().getAsInt())
-            .describedAs(description)
-            .isEqualTo(1);
+        assertThat(result.contains(COL_NAME)).describedAs(description).isTrue();
+        assertThat(result.getInt(COL_NAME)).describedAs(description).isEqualTo(1);
       }
 
       // for scan
-      for (Value<?> partitionKeyValue : partitionKeyValues) {
+      for (Column<?> partitionKeyValue : partitionKeyValues) {
         // Arrange
         Scan scan = prepareScan(partitionKeyType, partitionKeyValue);
 
         // Act Assert
         try (Scanner scanner = storage.scan(scan)) {
-          Optional<Result> result = scanner.one();
+          Optional<Result> optResult = scanner.one();
```

Comment:
[trivial] You used `resultOpt` at https://github.com/scalar-labs/scalardb/pull/2328/files#diff-d6e999e6390b76c30190d9b158a27634af73f3db28349b60430a5378bf681283R271. Using unified naming convention would be better? (Either of `resultOpt` or `optResult` is fine with me)

Reply:
Thank you! I will use `optResult`.
At first, I used `resultOpt` but changed it midway to `optResult` and forgot to rename that one.

Reply:
Revised in https://github.com/scalar-labs/scalardb/pull/2328/commits/4031a797c4ed7200c412367c1c4226c4eda1cdfe